src/main/java/uk/gov/hmcts/reform/prl/services/document/DocumentGenService.java

```diff
@@ -1118,27 +1096,23 @@ public Map<String, Object> generateDocumentsForTestingSupport(String authorisati
 
         documentLanguageIsEngForTestingSupport(authorisation, caseData, updatedCaseData, documentLanguage);
         documentLanguageIsWelshForTestingSupport(authorisation, caseData, updatedCaseData, documentLanguage);
-        if (documentLanguage.isGenEng() && !documentLanguage.isGenWelsh()) {
-            updatedCaseData.put(DOCUMENT_FIELD_FINAL_WELSH, null);
-            updatedCaseData.put(DOCUMENT_FIELD_C1A_WELSH, null);
-            updatedCaseData.put(DOCUMENT_FIELD_C8_WELSH, null);
-        } else if (!documentLanguage.isGenEng() && documentLanguage.isGenWelsh()) {
-            updatedCaseData.put(DOCUMENT_FIELD_FINAL, null);
-            updatedCaseData.put(DOCUMENT_FIELD_C8, null);
-            updatedCaseData.put(DOCUMENT_FIELD_C1A, null);
-        }
+        updatedCaseData.put(DOCUMENT_FIELD_FINAL_WELSH, null);
```

Comment:
We need to remove these lines I think. @khurshedk please take a look.

Reply:
@channa-patil removed as discussed